docs/events/timeline-ui-overview.asciidoc

```diff
@@ -202,7 +202,23 @@ The {ref}/esql.html[Elasticsearch Query Language ({esql})] provides a powerful w
 
 You can use {esql} in Timeline by opening the **{esql}** tab. From there, you can: 
 
-- Explore your events using the default query, or create a custom one. The default query searches documents within the Security alert index (`.alerts-security.alerts-default`) and indices specified in the <<update-sec-indices,Security data view>>, then returns 10 events from the defined time range.  
+- Write an {esql} query to explore your events. For example, start with the following query, then iterate on it to tailor your results: 
++
+[source,esql]
+----
+FROM .alerts-security.alerts-default,apm-*-transaction*,auditbeat-*,endgame-*,filebeat-*,logs-*,packetbeat-*,traces-apm*,winlogbeat-*,-*elastic-cloud-logs-*
+| LIMIT 10
+| KEEP @timestamp, message, event.category, event.action, host.name, source.ip, destination.ip, user.name
+----
++
+This query does the following:
+
+** It starts by querying documents within the Security alert index (`.alerts-security.alerts-default`) and indices specified in the <<update-sec-indices,Security data view>>.
+** Then, the query limits the output to the top 10 results.
+** Next, it keeps the `@timestamp`, `message`, `event.category`, `event.action`, `host.name`, `source.ip`, `destination.ip`, and `user.name` fields in the output.
+
+NOTE: An error message displays when the query bar is empty. 
```

Comment:
Would we also note here? "Without the keep clause, all fields for every index that matches the given `FROM` pattern will be requested. For any indices with a significant number of fields such as indices under `logs-*` this could lead to decreased performance of the query.

Not sure if this is the best place for that, but we should highlight that for users somewhere I think

Reply:
I can expand the note to include these performance notes.

Reply:
Took an initial stab at incorporating this information in [dbbff14](https://github.com/elastic/security-docs/pull/4578/commits/dbbff141b5cb1a4ac50a0c30a1a13dfbcb440e92) and [bf27e88](https://github.com/elastic/security-docs/pull/4578/commits/bf27e88bc613eaf00396833dffdd6e872f7c23c9).